src/main/kotlin/com/bugfender/UploadMappingPlugin.kt

```diff
@@ -19,7 +19,7 @@ class UploadMappingPlugin : Plugin<Project> {
                     project.logger.debug("[Bugfender] Attempting to apply plugin to {}", variant.name)
 
                     if (!variant.buildType.isMinifyEnabled) {
-                        project.logger.lifecycle("[Bugfender] Not applying to {}: minification is disabled", variant.name)
+                        project.logger.debug("[Bugfender] Not applying to {}: minification is disabled", variant.name)
```

Comment:
This sounds confusing because it looks like an error/warning. This just means BF is ignoring this build because there's no mapping file to upload. Therefore I moved it to debug level, where I think it belongs.